tests/unit/test_config.py

```diff
@@ -13,6 +13,11 @@ def _assert_config_read(environment, mock_config):
         mock_instance = mock_config.return_value
         Config.CONFIG = None  # Force config file reload
         prev_environment = os.environ.get(environment)
+        for to_remove in ['APPDATA', 'HOME', 'XDG_CONFIG_HOME']:
+            try:
+                del os.environ[to_remove]
```

Comment:
Should these be restored at the end of the test? I'm concerned about tests having an influence on one another since they'll all have different state when they start. `test_load_ini_with_no_config_directory` shows them being restored.

If not, then we can remove the `prev_environment` handling from both.